.github/workflows/release.yml

```diff
@@ -98,9 +102,9 @@ jobs:
           ls -lrt images/DAppNode-debian-bullseye-amd64.iso
 
       # Set new name for the release asset
-      - name: Set DAppNode unttended ISO name
+      - name: Set DAppNode unattended ISO name
         run: |
-          cp ./images/DAppNode-debian-bullseye-amd64.iso DAppNode-${CORE_VERSION}-debian-bullseye-amd64-unattended.iso
+          cp ./images/DAppNode-debian-bullseye-amd64.iso DAppNode-${CORE_VERSION}-${EXTRA_PKGS}-debian-bullseye-amd64-unattended.iso
```

Comment:
You are overwriting the standard release, instead you should create a new step to create the ISO with the extra packages so its a new asset to be added to the release

Reply:
Edit compose env on the fly